docs/plugins.md

```diff
@@ -390,3 +390,50 @@ Leaflet keeps it simple. If you can think of a feature that is not required by a
 There are no hard requirements on how to create your own plugin, but all developers are encouraged to read the recommendations in the [plugin guide](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md).
 
 Once your plugin is ready, you can submit it: just send a pull request with a new plugin file in [/docs/_plugins/](https://github.com/Leaflet/Leaflet/tree/main/docs/_plugins)to our GitHub repository.
+
+<script>
+function loadRepoData() {
+	const rows = document.querySelectorAll('.plugins table tr');
+	rows.forEach((row) => {
+		try {
+			const repoData = row.querySelector('.repo-data');
+			if (repoData) {
+				const link = row.querySelector('.plugin-repo-url').href;
+				let badges = [];
+				if (link.indexOf('github.com') > -1) {
```

Comment:
As below, but with ``` const regexpGithubCom = /^https?:\/\/(?:www)?\.github\.com\/([\w\d-_]+)\/([\w\d-_]+)\/?/; ```

Reply:
I don't get it work without `www`: `https://github.com/dekguh/real-usage-leaflet-movemarker`. Can you please help me